components/Card/CardTitle.tsx

```diff
@@ -29,7 +30,8 @@ export const CardTitle = (props: CardTitleProps) => {
     isUserMatch,
     subheader,
     type,
-    userRole
+    userRole,
+    isNewsfeed
```

Comment:
Should the generic `CardTitle` component have "Newsfeed"-specific properties? At first glance, it seems like we should have a generic field like "subheading" (I know, naming is tricky with the existing `subheader`) that the Newsfeed cards can set to the "You follow/don't follow ..." text and other cards can leave blank.

Reply:
Looking at this now, `CardTitle` is probably a little overstuffed with newsfeed-specific content (e.g. `isBillMatch`/`isUserMatch`/`authorUid`). 

Since we're supplying the header element directly in the Newsfeed cards anyway, would it make sense to extract the Newsfeed-specific content into a `NewsfeedCardTitle` (like we have the `*NewsfeedCardBody` components)? That way, `CardTitle` could continue to serve as a more generic fallback when we don't have specific header needs, but the `NewsfeedCard`s can keep the more complex layout we require.